db/migrate/20161109150714_add_templates_to_resolve_url.rb

```diff
@@ -0,0 +1,21 @@
+class AddTemplatesToResolveUrl < ActiveRecord::Migration[5.0]
```

Comment:
I think so.  I haven't yet figured out the straightforward way to get the previous version to the migration to test which is necessary to do in `before`, though.